installer/pkg/cmd/gcp_broker.go

```diff
@@ -443,3 +450,54 @@ func removeConfigs(dir string, filenames []string) error {
 	}
 	return nil
 }
+
+type createBrokerConfig struct {
+	name  string // name of the broker
+	title string // title of the broker
+}
+
+// NewCreateGCPBrokerCmd returns a cobra command which creates a new GCP service
+// broker without adding it to the existing Kubernetes cluster.
+func NewCreateGCPBrokerCmd() *cobra.Command {
+	cfg := &createBrokerConfig{
+		name:  "default",
+		title: "Default Broker",
```

Comment:
No need to assign default values twice (they will be assigned by the flags in line 474/475).

Reply:
done